docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '3.8'
+
+services:
+  redis:
+    image: redis:7 # Use the desired Redis version
```

Comment:
Let's use alpine images. They're lightweight. The latest stable version according to [docker hub](https://hub.docker.com/_/redis) is `7.4.1-alpine`